src/coprocessor/dag/expr/builtin_like.rs

```diff
@@ -0,0 +1,241 @@
+// Copyright 2018 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::i64;
```

Comment:
is it used?

Reply:
No, I will remove it, thank you

Reply:
@Hoverbear I'm wondering why there is no "unused" lint warnings for this..